target/linux/ramips/dts/HC5962.dts

```diff
@@ -0,0 +1,156 @@
+/dts-v1/;
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/input/input.h>
+
+/ {
+	model = "HiWiFi HC5962";
+
+	memory@0 {
+		device_type = "memory";
+		reg = <0x0 0x10000000>;
+	};
+
+	palmbus@1E000000 {
+		spi@b00 {
+			status="disabled";
+		};
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+	
+	gpio-leds {
+		compatible = "gpio-leds";
+
+		status-white {
+			label = "hc5962:white:status";
+			gpios = <&gpio0 6 0>;
+		};	
+		
+		system-red {
+			label = "hc5962:red:system";
+			gpios = <&gpio0 7 0>;
+		};
+	};
+
+	gpio-keys-polled {
+		compatible = "gpio-keys-polled";
+		#address-cells = <1>;
+		#size-cells = <0>;
+		poll-interval = <20>;
+
+		reset {
+			label = "reset";
+			gpios = <&gpio0 18 1>;
+			linux,code = <KEY_RESTART>;
+		};
+	};
+};
+	
+&nand {
+	status = "okay";
+
+	partition@0 {
+		label = "u-boot";
+		reg = <0x0 0x80000>;
+		read-only;
+	};
+
+	partition@80000 {
+		label = "debug";
+		reg = <0x80000 0x80000>;
+	};
+
+	factory: partition@100000 {
+		label = "factory";
+		reg = <0x100000 0x40000>;
+		read-only;
+	};
+	
+	firmware {
+		label = "firmware";
+		reg = <0x140000 0x2000000>;
+	};
+
+	partition@140000 {
+		label = "kernel";
+		reg = <0x140000 0x200000>;
+	};
+
+	partition@340000 {
+		label = "ubi";
+		reg = <0x340000 0x1E00000>;
+	};
+	
+	partition@2140000 {
+		label = "hw_panic";
+		reg = <0x2140000 0x80000>;
+	};		
+	
+	partition@21c0000 {
+		label = "bdinfo";
+		reg = <0x21c0000 0x80000>;
+	};
+	partition@2240000 {
+		label = "backup";
+		reg = <0x2240000 0x80000>;
+	};
+	
+	partition@22c0000 {
+		label = "overly";
```

Comment:
Are you sure about the name `overly`? Isn't it a typo?
And, same question as above (no read-only flag).